docs/components/modeler/bpmn/multi-instance/multi-instance.md

```diff
@@ -94,6 +94,14 @@ target: output
 
 A `completionCondition` defines whether the multi-instance body can be completed immediately when the condition is satisfied. It is a [boolean expression](/components/concepts/expressions.md#boolean-expressions) that will be evaluated each time the instance of the multi-instance body completes. Any instances that are still active are terminated and the multi-instance body is completed when the expression evaluates to `true`.
 
+The BPMN 2.0 specification defines the following properties of a multi-instance body instance:
+
+- `numberOfInstances`: The number of instances created.
+- `numberOfActiveInstances`: The number of instances currently active.
+- `numberOfCompletedInstances`: The number of instances already completed.
+
+These properties are available for use in the `completionCondition` expression. Although they are available in this expression, they do not exist as process variables. These properties take precedence over process variables with the same name.
```

Comment:
🔧 I'd like to see an additional sentence that describes what they can be used for. Perhaps we provide the example from below (if 50% of the instances have completed). WDYT?